addons/test_discuss_full/tests/test_avatar_card_tour.py

```diff
@@ -86,13 +86,36 @@ def setUpClass(cls):
                 "state": "validate",
             }
         )
+        cls.test_record = cls.env["mail.test.multi.company.with.activity"].create(
```

Comment:
`test_discuss_full` does not (and should not) depend on `test_mail` models. Just use a partner as thread or something like that.